sushi-config.yaml

```
@@ -38,101 +38,235 @@ dependencies:
+          valueCode: trial-use
     3-3-0-patientevents.md:
       title: Patient Events
+      extension:
+      - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+        valueCode: trial-use
       3-3-1-Patient-open.md:
         title: Patient-open Event
+        extension:
+        - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+          valueCode: trial-use
       3-3-2-Patient-close.md:
         title: Patient-close Event
+        extension:
+        - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+          valueCode: trial-use
     3-4-0-encounterevents.md:
       title: Encounter Events
+      extension:
+      - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+        valueCode: trial-use
       3-4-1-Encounter-open.md:
         title: Encounter-open Event
+        extension:
+        - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+          valueCode: trial-use
       3-4-2-Encounter-close.md:
         title: Encounter-close Event
+        extension:
+        - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+          valueCode: trial-use
     3-5-0-imagingstudyevents.md:
       title: ImagingStudy Events
+      extension:
+      - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+        valueCode: trial-use
       3-5-1-ImagingStudy-open.md:
         title: ImagingStudy-open Event
+        extension:
+        - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+          valueCode: trial-use
       3-5-2-ImagingStudy-close.md:
         title: ImagingStudy-close Event
+        extension:
+        - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+          valueCode: trial-use
     3-6-0-diagnosticreportevents.md:
       title: DiagnosticReport Events
+      extension:
+      - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+        valueCode: trial-use
       3-6-1-DiagnosticReport-open.md:
         title: DiagnosticReport-open Event
+        extension:
+        - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+          valueCode: trial-use
       3-6-2-DiagnosticReport-close.md:
         title: DiagnosticReport-close Event
+        extension:
+        - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+          valueCode: trial-use
       3-6-3-DiagnosticReport-update.md:
         title: DiagnosticReport-update Event
+        extension:
+        - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
+          valueCode: trial-use
       3-6-4-DiagnosticReport-select.md:
         title: DiagnosticReport-select Event
```

```suggestion
        title: DiagnosticReport-select Event
        extension:
        - url: http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status
          valueCode: trial-use
```
@bvdh , assuming this was an oversight and not intentional?